python/phase-2.md

```diff
@@ -923,6 +924,47 @@ What do we do? Here are the notes/options (activity options left off of calls fo
   This seems like a fairly sane way, but requires callers pass a special parameter to call
 * Another outstanding question - is there added benefit _at this time_ to support registering a class full of activity
   methods instead of individually? I am thinking not (yet).
+* Another option that was requested to be shown is to disallow `@activity.defn` on methods and make an easy way to
+  externally define those as callable. Here's the easiest way I can think of making that look:
+
+    ```python
+    class ActivityClass:
+        def __init__(self, some_init_arg: int) -> None:
+            pass
+
+        def some_activity1(self, arg: str) -> List[str]:
+            ...
+
+        def some_activity2(self, arg: int) -> List[int]:
+            ...
+
+    some_activity1 = workflow.activity_method_reference(ActivityClass.some_activity1)
+    some_activity2 = workflow.activity_method_reference(ActivityClass.some_activity2, name="some_other_name")
+
```

Comment:
Thanks for writing this. Personally, I don't mind it, but I can see how people would get annoyed by it for sure. Given that the other approach does type-level prevent you from using the wrong call for the type of activity, I'm happy with that.